front/src/modules/companies/components/AddPersonToCompany.tsx

```diff
@@ -12,14 +13,32 @@ import { GET_PEOPLE } from '@/people/graphql/queries/getPeople';
 import { ButtonSize } from '@/ui/button/components/Button';
 import { IconButton } from '@/ui/button/components/IconButton';
 import { RelationPickerHotkeyScope } from '@/ui/input/relation-picker/types/RelationPickerHotkeyScope';
+import { TextInput } from '@/ui/input/text/components/TextInput';
 import { usePreviousHotkeyScope } from '@/ui/utilities/hotkey/hooks/usePreviousHotkeyScope';
 import { RecoilScope } from '@/ui/utilities/recoil-scope/components/RecoilScope';
-import { useUpdateOnePersonMutation } from '~/generated/graphql';
+import {
+  useInsertOnePersonMutation,
+  useUpdateOnePersonMutation,
+} from '~/generated/graphql';
 
 const StyledContainer = styled.div`
   position: relative;
 `;
 
+const StyledInputContainer = styled.div`
+  background-color: ${({ theme }) => theme.background.tertiary};
+  box-shadow: ${({ theme }) => theme.boxShadow.strong};
+  display: flex;
+  gap: ${({ theme }) => theme.spacing(0.5)};
+  width: ${({ theme }) => theme.spacing(62.5)};
+  & * input,
+  * div {
```

Comment:
The `StyledTrailingIconContainer` in `TextInput.tsx` causes an extra div to appear, these changes override that behavior and changes the input background instead of crafting a new styled component just for this.

Reply:
Ok, we might have to rework the input instead. I'm merging as it is but we will re-work the inputs next week